pyuvdata/uvcal/initializers.py

```diff
@@ -0,0 +1,529 @@
+    time_array : ndarray of float
+        Array of times in Julian Date.
+    antenna_positions : ndarray of float or dict of ndarray of float
+        Array of antenna positions in ECEF coordinates in meters.
+        If a dict, keys can either be antenna numbers or antenna names, and values are
+        position arrays. Keys are interpreted as antenna numbers if they are integers,
+        otherwise they are interpreted as antenna names if strings. You cannot
+        provide a mix of different types of keys.
+    telescope_location : ndarray of float or str
+        Telescope location as an astropy EarthLocation object or MoonLocation object.
+    telescope_name : str
+        Telescope name.
+    cal_style : str
+        Calibration style. Options are 'sky' or 'redundant'.
+    gain_convention : str
+        Gain convention. Options are 'divide' or 'multiply'.
+    x_orientation : str
+        Orientation of the x-axis. Options are 'east', 'north', 'e', 'n', 'ew', 'ns'.
+    jones_array : ndarray of int or str
+        Array of Jones polarization integers. If a string, options are 'linear' or
+        'circular' (which will be converted to the appropriate Jones integers as
+        length-4 arrays).
+    delay_array : ndarray of float, optional
+        Array of delays in seconds. If cal_type is not provided, it will be set to
+        'delay' if delay_array is provided.
+    cal_type : str, optional
+        Calibration type. Options are 'delay', 'gain', or 'unknown'. Not required
+        if delay_array is provided (then it will be set to 'delay').
+    integration_time : float or ndarray of float, optional
+        Integration time in seconds. If not provided, it will be set to the minimum
+        time separation in time_array for all times.
+    channel_width : float or ndarray of float, optional
+        Channel width in Hz. If not provided, it will be set to the difference
+        between channels in freq_array (with the last value repeated).
+    antenna_names : list of str, optional
+        List of antenna names. If not provided, antenna numbers will be used to form
+        the antenna_names, according to the antname_format. antenna_names need not be
+        provided if antenna_positions is a dict with string keys.
+    antenna_numbers : list of int, optional
+        List of antenna numbers. If not provided, antenna names will be used to form
+        the antenna_numbers, but in this case the antenna_names must be strings that
+        can be converted to integers. antenna_numbers need not be provided if
+        antenna_positions is a dict with integer keys.
+    antname_format : str, optional
+        Format string for antenna names. Default is '{0:03d}'.
+    ant_array : ndarray of int, optional
+        Array of antenna numbers actually found in data (in the order of the data
+        in gain_array etc.)
+    wide_band : bool, optional
+        Whether to use wide_band. Default is False, and Trie is not yet supported.
```

Comment:
Trie -> True

wide_band will have to be true if it's a delay style cal, right?

Reply:
Actually, I just forgot to remove this from the argument list. It's set automatically in the code depending on whether ``freq_array`` or ``freq_range`` is provided (False and True respectively).